src/main/java/uk/gov/hmcts/darts/retention/enums/Policy.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.darts.retention.enums;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum Policy {
+    PERMANENT("08170758956b0614","DARTS Permanent Retention v3"),
```

Comment:
You don't need the 'policyNumber' field in the code. They are for migrated data.

Reply:
policyNumber removed